deployment/terraform/cloud-config/ecs-batch-user-data.yml

```diff
@@ -3,19 +3,22 @@
 packages:
     - awslogs
 
-fs_setup:
-  - label: nvme0n1
-    filesystem: ext4
-    extra_opts: ["-E", "nodiscard"]
-    device: /dev/nvme0n1
-    partition: auto
-
-mounts:
-  - [ /dev/nvme0n1, null ]
-  - [ /dev/nvme0n1, "/media/nvme0n1", "ext4", "defaults,nobootwait,discard", "0", "2" ]
-
 runcmd:
   - curl -o /etc/papertrail-bundle.pem https://papertrailapp.com/tools/papertrail-bundle.pem
+  # Manually mount i3 family ephemeral storage, see:
+  # https://bugs.launchpad.net/cloud-init/+bug/1672833
+  # o - Create empty dos partition table
+  # n - Add new partition
+  # p - Primary partition (In add new prompt)
+  # 1 - Partition number (In add new prompt)
+  # empty newline - First sector, accepts default: 1
+  # empty newline - Last sector, accepts default: end
+  # w - Write changes
+  - echo -e "o\nn\np\n1\n\n\nw" | fdisk /dev/nvme0n1
+  - mkfs.ext4 /dev/nvme0n1p1
```

Comment:
Based on (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html), I think that we may want this to be:

```yaml
- mkfs.ext4 -E nodiscard /dev/nvme0n1p1
```